lib/features/composer/presentation/composer_controller.dart

```diff
@@ -1700,6 +1706,20 @@ class ComposerController extends BaseController with DragDropFileMixin {
     }
   }
 
+  void _onScrollToBottomListener() {
+    if (scrollController.position.atEdge // screen at top or bottom edge
+      && scrollController.position.pixels > 0 // screen not at top 
+      && !_isEmailBodyLoaded // email not loaded
+      && !ToastView.isVisible // no toast showing
```

Comment:
IMO, We should check the correct toast of the scroll. Because there may be a few other toasts showing at that time.

Reply:
IMO, this toast is less priority than other toast. So it is ok to disable this toast when other toast still visible.

Reply:
We don't need it anymore. We have fixed the cut email error